sig-release/charter.md

```diff
@@ -14,10 +14,11 @@ This charter adheres to the conventions described in the [Kubernetes Charter REA
 - Ensuring quality Kubernetes releases
   - Defining and staffing release roles to manage the resolution of release blocking criteria
   - Defining and driving development processes (e.g. merge queues, cherrypicks) and release processes
-    (e.g. burndown meetings, cutting beta releases) with the intent of meeting the release schedule
+    (e.g. burndown meetings, cutting pre-releases) with the intent of meeting the release schedule
   - Managing the creation of release specific artifacts, including:
```

Comment:
I am not sure this is really the place for it, but did want to call out that the original review that led to opening this issue requested more clarity on the specific binaries, tools, repos, and artifacts that fell under the purview of sig-release. There is a bit of a fuzzy line because almost everything sig-release releases is "owned" by another sig. I plan on having the specific artifacts better documented as part of https://github.com/kubernetes/sig-release/issues/1337, but wanted to make sure @spiffxp knows that is not falling through the cracks.

ref https://github.com/kubernetes/community/pull/2919#discussion_r232103575 and https://github.com/kubernetes/community/pull/2919#discussion_r232104680

Reply:
I think having a specific list of artifacts better documented via https://github.com/kubernetes/sig-release/issues/1337 satisfies most of my original concern.  That said, https://github.com/kubernetes/community/pull/2919#discussion_r232104680 still stands, I would like to see enumeration of what release-engineering owns, but that can also be done out-of-charter and linked

IMO there is overlap on driving development processes w/ SIG Architecture, SIG Contributor Experience and SIG Testing but I'm not concerned enough about the wording here to block